server/controllers/project.controller/createProject.js

```diff
@@ -9,7 +9,7 @@ export default function createProject(req, res) {
   projectValues = Object.assign(projectValues, req.body);
 
   function sendFailure() {
-    res.json({ success: false });
+    res.status(400).json({ success: false });
```

Comment:
Decided to change this to `400` over `422` because it was more general? It would be really nice to get more granular with the error codes but since this function was called to handle different errors I thought `400` would be more appropriate.